cmake-modules/AddGoogleTest.cmake

```diff
@@ -78,7 +78,8 @@ macro(add_gtest TESTNAME)
         else()
             gtest_discover_tests(${TESTNAME}
                 TEST_PREFIX "${TESTNAME}."
-                PROPERTIES FOLDER "Tests")
+                PROPERTIES FOLDER "Tests"
+                DISCOVERY_TIMEOUT 600)
```

Comment:
Looks like the default is 5. Is 600 seconds too log?

Reply:
Rick said make it 10 minutes. I'm making it 10 minutes.